tests/test_regression/TestControllers.py

```diff
@@ -80,6 +80,34 @@ def test_delete(self):
             )
             self.assertEqual(response.status_code, 302) # 302 is for Redirection
 
+    def test_delete_v2(self):
```

Comment:
What does v2 mean?

Reply:
i guess he mean version 2. @T1duS change the function name to more meaningful.

Reply:
@satyammittal Yes, what's version 2? What's version 1? Why we have two versions?

Reply:
there are 2 test_delete function, see the function above (line number 55). we can remove the above function. and keep only version 2.